docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -639,17 +639,11 @@ Standard Arming
 ==========================
 DroneCAN can use the same advanced arming procedure as all other throttle sources. The details of this arming procedure are covered in the :ref:`manual_advanced_arming` section.
 
+.. _arm_with_armingstatus:
+
 Arming with ArmingStatus
 ==========================
 
-.. note::
-	In order to control your module properly while arming with ArmingStatus messages, you must properly set the :ref:`manual_arming_throttle_source` parameter. This parameter should 
```

Comment:
doesn't exist any more

Reply:
We still have stuff about the manual arming throttle source thing in some legacy section right? Just checking its not  yet been totally purged